lib/cluster.js

```diff
@@ -341,9 +341,9 @@ function masterInit() {
       process: workerProcess
     });
 
-    worker.on('message', (message, handle) =>
-      cluster.emit('message', message, handle)
-    );
+    worker.on('message', function(message, handle) {
+      cluster.emit('message', this, message, handle);
```

Comment:
Even I was thinking about the closure variable access. But then every time someone goes through the code, they would have to spend time on this to really understand this. Maybe that's just me.